readme.md

```diff
@@ -15,9 +15,11 @@
 [npm][]:
 
 ```sh
-npm install rehype-autolink-headings
+npm install rehype-autolink-headings rehype-slug
 ```
 
+> **This package will not work without `rehype-slug`. More information under [API](#user-content-api).**
```

Comment:
A more accurate version of this is noted in https://github.com/rehypejs/rehype-autolink-headings#rehypeuselink-options

If wanted this note could be moved up higher in the readme.

```suggestion
> Note: this plugin expects `id`s to already exist on headings. One way to add those automatically, is [rehype-slug](https://github.com/rehypejs/rehype-slug) (see example).
```